### ISSUE TITLE
DRAFT: Provision cloudfront/s3/route 53 for vector tile serving

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#764]
# Draft

This is a draft issue, and it will need to clarify specs and formalize what
counts as "done" before work begins on it.

Check with the issue's creator before starting work on this issue.

## Overview

Required by #763

### Describe the solution you'd like

- Add terraform provisioning code to create a system like Model My Watershed. Cloudfront targeting an S3 bucket which falls back to the app tile endpoint (the app will write tiles back to S3 in #763)
- Forward original referrer and custom header to the tile endpoint if at all possible.

### Additional Context

Optional. Add any other context, screenshots, gifs, etc about the feature request here.

**Comment by rajadain** [comment]
Superseded by #779.
